### PR TITLE
Intellisense:  Allow specification of full paths.

### PR DESCRIPTION
**Pull request by scotthal**
Make sure these boxes are checked before submitting your PR -- thanks in advance!

- [X] Build pass!
- [X] Follow VS Code [Coding Guidelines](https://github.com/Microsoft/vscode/wiki/Coding-Guidelines)

Allow specification of full path to the Ruby interpreter and
rct-complete command for environments in which multiple versions
of Ruby and gems are installed.

### CODE REVIEW

File: ruby.js [hunk 1]
```diff
@@ -123,9 +123,12 @@ function balanceEvent(event) {
 }
 
 function completeCommand(args) {
+	let rctCompletePath = vscode.workspace.getConfiguration('ruby.rctComplete').commandPath;
+	args.push('--interpreter');
+	args.push(vscode.workspace.getConfiguration('ruby.interpreter').commandPath);
 	if (process.platform == 'win32')
```
Comment (rebornix) [inline ruby.js]: I suppose if we don't set a default value for this option, the value we get is an empty string, which makes this line of code not intuitive. Maybe pass in the default value?

Comment (scotthal) [inline ruby.js]: Done.

**Comment by rebornix** [comment]
@scotthal thanks for your quick update! Love it!

**Comment by danipindado** [comment]
how is this supposed to work for windows? 
i have tried
"ruby.interpreter.commandPath": "C:/Ruby24-x64/bin/ruby.exe"
"ruby.interpreter.commandPath": "C:\\Ruby24-x64\\bin\\ruby.exe"
and many more combinations. but in the end the ruby version present in PATH env. variable is executed.

**Pull request merged**